app/assets/html/survey_questions.html

```diff
@@ -1,3 +1,5 @@
+<div ng-include src="'/assets/progress_bar.html'"></div>
+
```

Comment:
I must prefer creating two ui-views, and setting the controllers and templates in the router.coffee.